Makefile

```diff
@@ -172,9 +172,18 @@ bin/Test$(CONFIGURATION)/$(JAVA_INTEROP_LIB): bin/$(CONFIGURATION)/$(JAVA_INTERO
 run-android: $(ATESTS)
 	(cd src/Android.Interop/Tests; $(MSBUILD) $(MSBUILD_FLAGS) '/t:Install;RunTests' $(if $(FIXTURE),/p:TestFixture=$(FIXTURE)))
 
-run-test-jnimarshal: bin/Test$(CONFIGURATION)/Java.Interop.Export-Tests.dll bin/Test$(CONFIGURATION)/$(JAVA_INTEROP_LIB)
+JRE_DLL_CONFIG_SRC=src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config
+JRE_DLL_CONFIG=bin/$(CONFIGURATION)/Java.Runtime.Environment.dll.config
+
+$(JRE_DLL_CONFIG): src/Java.Runtime.Environment/Java.Runtime.Environment.csproj
+	$(MSBUILD) $(MSBUILD_FLAGS) $<
+
+run-test-jnimarshal: bin/Test$(CONFIGURATION)/Java.Interop.Export-Tests.dll bin/Test$(CONFIGURATION)/$(JAVA_INTEROP_LIB) $(JRE_DLL_CONFIG_SRC) $(JRE_DLL_CONFIG)
+	mv -fv $(JRE_DLL_CONFIG) $(JRE_DLL_CONFIG).bak
```

Comment:
Why does `$(JRE_DLL_CONFIG)` need to be backed up? I don't see anything here which would alter its contents.

Additionally, filenames should always be quoted.

Reply:
Because we want to restore it once the test is run.